packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -635,6 +654,90 @@ export const SharedTreeFormatVersion = {
  */
 export type SharedTreeFormatVersion = typeof SharedTreeFormatVersion;
 
+/**
+ * Defines for each EditManagerFormatVersion the SharedTreeChangeFormatVersion to use.
+ * This is an arbitrary mapping that is injected in the EditManger codec.
+ * Once an entry is defined and used in production, it cannot be changed.
+ * This is because the format for SharedTree changes are not explicitly versioned.
+ */
+export const changeFormatVersionForEditManager: DependentFormatVersion<
+	EditManagerFormatVersion,
+	SharedTreeChangeFormatVersion
+> = DependentFormatVersion.fromPairs([
+	// [EditManagerFormatVersion, SharedTreeChangeFormatVersion]
+	[1, 1],
+	[2, 2],
+	[3, 3],
+	[4, 4],
+	[5, 4],
```

Comment:
this looks backwards somehow--should be SharedTreeChangeFormatVersion 5 corresponds to EditManager version 4.

It feels like there's duplication between the information here and in `formatVersionToTopLevelCodecVersions`--can we remove it?

Reply:
ah, yes. that is what was happening. Thanks!